### DISCUSSION TITLE
Inconsistency on what I build in builder and on preview page

### DISCUSSION THREAD

**Discussion by rzkmsjhr** [Budibase/budibase#6323] [Help] [answered]
My app is not showing what changes that I've made on my builder, what makes this an issue?

![Screenshot 2022-06-15 080955](https://user-images.githubusercontent.com/11987529/173715874-73826b8a-511d-478e-bc69-63396e66dec2.png)
![Screenshot 2022-06-15 081008](https://user-images.githubusercontent.com/11987529/173715878-f704bd71-da67-4cca-9bf5-89bb49b7c316.png)

**Comment by melohagan** [comment]
Interesting, I see the elements are present, it's the formatting that differs. 
Maybe because the preview is wider it affects the presentation.

Are you using any custom CSS? 

Also, it may be worth trying an upgrade - the blue cycle icon near the top right indicates an update is available. Just in case that works!

**Reply by rzkmsjhr** [reply]
Thanks for the reply, I've tried your solution but still no effect. I'm not using any custom css.

**Reply by melohagan** [reply]
Would you be able to send your app export to support@budibase.com please? I can take a look.

**Comment by rzkmsjhr** [comment, marked as answer]
somehow it back to normal after you export-import it.
